docs/tutorials/configure/build-a-mock-robot.md

```diff
@@ -25,42 +25,62 @@ This tutorial will show you how to build a mock robot using just your personal l
+- Typically, simple robots have just one part, but you can have as many parts as your project requires.
+- Parts are organized in a tree, with one of them being the _main_ part, and the others being _sub-parts_.
+- You can access any sub-part either directly, or through any part above it in the tree.
+- Each part runs a single `viam-server` instance.
+
+### What is a `fake` component?
+
+Most Viam [components](../../../components/) have a `fake` model that you can use for testing.
+These `fake` components interact with Viam like real hardware but do not actually exist.
+
+## Set up a mock robot
 
 You'll need the following hardware and software for this tutorial:
 
 - A laptop or desktop running Linux or macOS
 - [Go](https://go.dev/doc/install) or [Python 3.9+](https://www.python.org/downloads/)
 - A code editor of your choice
 
-## Set up a mock robot
-
 ### Make an account on the Viam app
 
-The first thing you need to do is set up your account on the Viam app.
-Go to [app.viam.com](https://app.viam.com) and sign up for a free account.
+First, set up your account on the Viam app.
+Go to [app.viam.com](https://app.viam.com) and sign up for an account.
+
+### Install and start `viam-server` on your computer
+
+Before you proceed with configuring and controlling your robot, install `viam-server`.
+
+Follow the steps outlined for your computer's architecture on the **Setup** tab of the [Viam app](https://app.viam.com) to install `viam-server` on your computer as a system service.
+
+Find more information on running `viam-server` in the [installation guide](/installation/manage/).
 
 ### Configure your mock robot
 
-Now you'll [configure your robot](/manage/configuration/) to represent your robot's hardware.
-If you were using actual hardware, this file would tell your code what hardware is attached to it and how to communicate with it.
+Next, [configure your robot](/manage/configuration/) to represent your robot's hardware.
 
-Since this is an imaginary robot, you will use `fake` components so that the Viam software ([`viam-server`](../../../viam/#get-started)) doesn't try and fail to connect to physical hardware.
+If you were using actual hardware, this process would tell `viam-server` what hardware is attached to it and how to communicate with it.
+Since you don't have physical hardware, you will use `fake` components.
 
-1. Go to [app.viam.com](https://app.viam.com/).
-2. Create a new [robot](../../../manage/fleet/robots/).
-3. Go to the new robot's **Config** tab.
-4. Configure a [fake board component](../../../components/board/fake/):
+1. Create a new [robot](/manage/fleet/robots/) in your organization and location on [app.viam.com](https://app.viam.com/).
+2. Go to the new robot's **Config** tab.
+3. Configure a [fake board component](/components/board/fake/):
 
     - Create a new component called `myBoard` with **Type** `board` and **Model** `fake`.
     Click **Create component**.
```

Comment:
I'm not touching this tutorial in the config UI ticket since I know it's being refactored here and don't want to make lots of merge conflicts, so mind updating these config instructions to reflect the updated UI as part of this PR?

Reply:
will do! Will comment here when done

Reply:
Thank you!

Reply:
Done I think!